index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
         </ul>
       </section>
     </section>
+
+<section class="whole-middle-section">
+    <section class="recent-work">
+        <h1 class="recent-work-content">
+            My Recent Works
+        </h1>
```

Comment:
- [ ] Well done on adding the heading for the middle section but kindly decrease it to a lower hx heading because following semantic HTML on one page it should be only one h1 and it should be in the beginning. Keep coding 💪

Reply:
Okay, I have done this change.